examples/gno.land/p/demo/markdown/markdown.gno

```diff
@@ -0,0 +1,126 @@
+package markdown
+
+import (
+	"strings"
+
+	"gno.land/p/demo/ufmt"
+)
+
+// Bold returns bold text for markdown
+func Bold(text string) string {
+	return ufmt.Sprintf("**%s**", text)
+}
+
+// Italic returns italicized text for markdown
+func Italic(text string) string {
+	return ufmt.Sprintf("*%s*", text)
+}
+
+// Strikethrough returns strikethrough text for markdown
+func Strikethrough(text string) string {
+	return ufmt.Sprintf("~~%s~~", text)
+}
+
+// H1 returns a level 1 header for markdown
+func H1(text string) string {
+	return ufmt.Sprintf("# %s", text)
+}
+
+// H2 returns a level 2 header for markdown
+func H2(text string) string {
+	return ufmt.Sprintf("## %s", text)
+}
+
+// H3 returns a level 3 header for markdown
+func H3(text string) string {
+	return ufmt.Sprintf("### %s", text)
+}
+
+// BulletList returns an bullet list for markdown
+func BulletList(items []string) string {
+	var sb strings.Builder
+	for _, item := range items {
+		sb.WriteString(ufmt.Sprintf("- %s\n", item))
+	}
+	return sb.String()
+}
+
+// OrderedList returns an ordered list for markdown
+func OrderedList(items []string) string {
+	var sb strings.Builder
+	for i, item := range items {
+		sb.WriteString(ufmt.Sprintf("%d. %s\n", i+1, item))
+	}
+	return sb.String()
+}
+
+// TodoItem represents a single todo item
+type TodoItem struct {
+	Item string
+	Done bool
+}
+
+// TodoList returns a list of todo items with checkboxes for markdown
+func TodoList(items []TodoItem) string {
```

Comment:
not sure if it would be better, but it could be nice to find a solution that would avoid having to use custom types and avoid something like this:

```go
tl := markdown.TodoList([]markdown.TodoItem{ // this line is the one I dislike
	{},
	{},
	// ...
})
```

Reply:
0aaf9110d85242ae744aafdfb13d5ef0cc76a89e do u like this change ?

Reply:
changed
`func TodoList(items []TodoItem) string `
to
`func TodoList(items []string, done []bool) string `